src/prices/errors/errors.ts

```diff
@@ -0,0 +1,10 @@
+export const PRICES_ERROR = {
+  RESPONSE_ERROR: {
+    name: 'ResponseError',
+    code: '2_1',
+  },
+  INVALID_DATE_ERROR: {
+    name: 'InvalidDateError',
+    code: '2_2',
```

Comment:
not sure if these `2_2` codes is your error code format that you want to use? You can also use `'INVALID_DATE_ERROR'` as an error code if you want